atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java

```diff
@@ -207,7 +205,7 @@ private boolean isHostHealthy(InetSocketAddress host) {
         try {
             CassandraClientPoolingContainer testingContainer = new CassandraClientPoolingContainer(host, config);
             testingContainer.runWithPooledResource(describeRing);
-            testingContainer.runWithPooledResource(CassandraVerifier.healthCheck);
+            testingContainer.runWithPooledResource(validatePartitioner);
```

Comment:
healthCheck is very very basic and can only fail if the node is truely buggered.

validatePartitioner is strictly speaking a superset of functionality testing.